tests/meta/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ GET_PYTHON_ENV()
 # working directory of tester should be binary dir so link data folders
 IF (NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/reference_results)
 	IF (NOT EXISTS ${CMAKE_SOURCE_DIR}/data/testsuite/meta/)
-		MESSAGE(FATAL_ERROR "Test suite for meta examples does not exist!")
+		MESSAGE(FATAL_ERROR "Test suite for meta examples does not exist! please enter `git submodule update --init` to download. ")
```

Comment:
Make this: "Please initialise and/or update the data submodule."

It is never a good idea to put explicit commands/names/etc into messages for users as these things outdate at lightning speed.